index7.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style7.css">
+    <title>hw7</title>
+</head>
+<body>
+    <header>
+        <h2>WHAT OUR CUSTOMERS ARE SAYING</h2>
+        <hr>
+        <h5>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</h5>
+    </header>
+    <main class="mid">
```

Comment:
I would personally name classes based on the content. For example, this section contains product reviews, therefore I would name it "reviews"